[2025-09-02T21:16:15.085+0800] INFO - local_task_job_runner.py:123 - _execute() - ::group::Pre task execution logs
[2025-09-02T21:16:15.232+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_parallel_dag.task5 scheduled__2025-09-02T12:00:00+00:00 [queued]>
[2025-09-02T21:16:15.295+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_parallel_dag.task5 scheduled__2025-09-02T12:00:00+00:00 [queued]>
[2025-09-02T21:16:15.296+0800] INFO - taskinstance.py:2866 - _check_and_change_state_before_execution() - Starting attempt 1 of 2
[2025-09-02T21:16:15.906+0800] INFO - taskinstance.py:2889 - _check_and_change_state_before_execution() - Executing <Task(PythonOperator): task5> on 2025-09-02 12:00:00+00:00
[2025-09-02T21:16:15.941+0800] INFO - standard_task_runner.py:72 - _start_by_fork() - Started process 264 to run task
[2025-09-02T21:16:15.997+0800] INFO - standard_task_runner.py:104 - _start_by_fork() - Running: ['***', 'tasks', 'run', 'example_parallel_dag', 'task5', 'scheduled__2025-09-02T12:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/example_parallel_dag.py', '--cfg-path', '/tmp/tmpucebr6ja']
[2025-09-02T21:16:16.002+0800] INFO - standard_task_runner.py:105 - _start_by_fork() - Job 11: Subtask task5
[2025-09-02T21:16:16.975+0800] INFO - task_command.py:467 - task_run() - Running <TaskInstance: example_parallel_dag.task5 scheduled__2025-09-02T12:00:00+00:00 [running]> on host a246a11defc8
[2025-09-02T21:16:18.664+0800] INFO - taskinstance.py:3132 - _execute_task_with_callbacks() - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='example_parallel_dag' AIRFLOW_CTX_TASK_ID='task5' AIRFLOW_CTX_EXECUTION_DATE='2025-09-02T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-02T12:00:00+00:00'
[2025-09-02T21:16:18.682+0800] INFO - taskinstance.py:731 - _execute_callable() - ::endgroup::
[2025-09-02T21:16:18.867+0800] INFO - logging_mixin.py:190 - _propagate_log() - Task task5 開始執行...
[2025-09-02T21:16:23.883+0800] INFO - logging_mixin.py:190 - _propagate_log() - Task task5 執行完成！
[2025-09-02T21:16:23.884+0800] INFO - python.py:240 - execute() - Done. Returned value was: None
[2025-09-02T21:16:24.069+0800] INFO - taskinstance.py:340 - _run_raw_task() - ::group::Post task execution logs
[2025-09-02T21:16:24.086+0800] INFO - taskinstance.py:352 - _run_raw_task() - Marking task as SUCCESS. dag_id=example_parallel_dag, task_id=task5, run_id=scheduled__2025-09-02T12:00:00+00:00, execution_date=20250902T120000, start_date=20250902T131615, end_date=20250902T131624
[2025-09-02T21:16:24.401+0800] INFO - local_task_job_runner.py:266 - handle_task_exit() - Task exited with return code 0
[2025-09-02T21:16:24.478+0800] INFO - local_task_job_runner.py:245 - _execute() - ::endgroup::
